db/db_iter.cc

```diff
@@ -644,6 +644,7 @@ void DBIter::ReverseToBackward() {
     while (iter_->Valid() &&
            user_comparator_->Compare(ikey.user_key, saved_key_.GetUserKey()) >
                0) {
+      assert(ikey.sequence != kMaxSequenceNumber);
       if (ikey.sequence > sequence_) {
```

Comment:
The warning was that in line 648 ikey.sequence can be unitialized. I hacked it to always initialize it to make the tool happy.